docs/qprofiler2/index.md

```diff
@@ -0,0 +1,326 @@
+                                only output HD and SQ.                                                  
+--bam-index                   Opt, a BAI index file. The option of "threads-producer" will be ignored   
+                                if no index file provided.                                              
+--bam-records <Integer>       Opt, only process the specified number of records from the beginning of   
+                                the BAM file. Default to process entire BAM file.                       
+--bam-validation              Opt, BAM record validation stringency [STRICT,LENIENT,SILENT].            
+                                Def=LENIENT.                                                            
+--help                        Show usage and help.                                                      
+--input                       Req, input file in FASTQ, BAM/SAM or VCF format, also support FA, MA,     
+                                QUAL, FASTA, GFF and GFF3 format.                                       
+--log                         Req, log file.                                                            
+--loglevel                    Opt, logging level [INFO,DEBUG], Def=INFO.                                
+--output                      Opt, XML output which containing basic summary statistics. Def="qprofiler.
+                                xml".                                                                   
+--threads-consumer <Integer>  Opt, number of threads to process inputed records, BAM and FASTQ mode     
+                                only. Def=0 (not multi-thread mode).                                    
+--threads-producer <Integer>  Opt, number of threads to read the indexed BAM file, BAM mode only. This  
+                                option will be ignored without "threads-consumer" option.  Def=1.       
+--vcf-format-field            Opt, group VCF records according to user specified format fields. eg.     
+                                "GD", "CCM","FT=PASS". Default to ignore VCF format field.              
+--version                     Show version number.     
+~~~~
+
+
+### multi thread
+When running multi-threaded, we suggest more consumer than producer threads with a recommended ratio of 6:1. For example, only two threads to read (producer) the input file but 12 threads are specified to process reads (consumer). The BAM index file should also be specified in multi-producer mode. eg. 
+
+~~~~{.text}
+java -jar qprofiler2.jar --threads-consumer 12 -threads-producer 2 --bam-index  $somedir/${bam}.bai --input  $somedir/$bam --output $somedir/${bam}.qp2.xml --log $somedir/${bam}.qp2.log
+~~~~
+
+## Output
+
+### Xml Validation
+
+qprofiler2 provides a schema file which help you to validate the xml output. This xsd file can download from https://purl.org/adamajava/xsd/qprofiler2/v2/qprofiler2.xsd
```

Comment:
qprofiler2 provides a schema file which helps you to validate the xml output. This xsd file can be download from https://purl.org/adamajava/xsd/qprofiler2/v2/qprofiler2.xsd